src/components/atoms/service-apps-data/ServiceAppsData.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react"
+import "./ServiceAppsData.scss";
+
+interface ServiceAppsDataProps{
+  username?: string
+  password?: string
+  notes?: string
+  lastModified?: string
+  created?: string
+  editMode: boolean
+}
+
+const ServiceAppsData = (props : ServiceAppsDataProps) => {
+  const [mode, setMode] = useState(props.editMode);
+
+  const editClick = () =>{
+    setMode(true);
+  }
+  const cancelClick = () =>{
+    setMode(false);
+  }
+
+  if(mode) {
+    return (
+      <div className='ServiceAppsData'>
+        <button className='btn-base save-btn'>Save</button>
+        <button className='btn-base delete-btn'>Delete</button>
+        <button className='btn-base cancel-btn' onClick={cancelClick}>Cancel</button>
```

Comment:
A base button component already exists, if you feel motivated can create the variants components following this practice:

https://blog.bitsrc.io/design-systems-react-buttons-with-the-base-variant-pattern-c56a3b394aaf